javatests/arcs/core/host/TestConstructedParticle.kt

```diff
@@ -0,0 +1,23 @@
+package arcs.core.host
+
+import arcs.core.data.Plan
+import arcs.core.data.Schema
+import arcs.core.entity.EntityBaseSpec
+import arcs.sdk.BaseParticle
+import arcs.sdk.HandleHolderBase
+
+fun build(spec: Plan.Particle): TestConstructedParticle {
```

Comment:
Can you come up with a better name? TestConstructedParticle doesn't say much. If we like the "reflective" name from the above comment then I would go for TestReflectiveParticle.